### PR TITLE
Update aqua doc

### PR DESCRIPTION
**Pull request by hrcornejo**

### CODE REVIEW

**Review by metmajer — APPROVED** [review]
Thanks @hrcornejo 💯

File: docs/modules/jenkins-shared-library/partials/odsComponentStageScanWithAqua.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ As a result, a Bitbucket Code Insight entry is added to the git commit (in Bitbu
 contains a link to the scan result on the Aqua platform. The Bitbucket Code Insight entry can be seen in a pull request.
 The pull request in Bitbucket shows the Code Insight of the latest commit of the PR.
 
-To get started, make sure you have a `ConfigMap` in your project-cd namespace in OpenShift that has these fields:
+To get started, make sure you have a `ConfigMap` in OpenDevStack project namespace (usually ods) in OpenShift that has these fields:
```
Comment (gerardcl) [inline docs/modules/jenkins-shared-library/partials/odsComponentStageScanWithAqua.adoc:9]: I would add which should be the expected name of the cm (`aqua` AFAIK, right?)

**Review by gerardcl — APPROVED** [review]
LGTM! 🚀 
I just added a minor comment to consider 👍

File: docs/modules/jenkins-shared-library/partials/odsComponentStageScanWithAqua.adoc (hunk 1 above)
Comment (hrcornejo) [reply docs/modules/jenkins-shared-library/partials/odsComponentStageScanWithAqua.adoc:9]: Yes, in the example is indicated that the name is aqua

Comment (gerardcl) [reply docs/modules/jenkins-shared-library/partials/odsComponentStageScanWithAqua.adoc:9]: yes! saw it but just suggesting to make it explicit when using UI

**Pull request merged**